src/input_xml.F90

```diff
@@ -5946,13 +6018,14 @@ subroutine read_0K_elastic_scattering(nuc, libraries, library_dict)
         ! Read nuclide data from HDF5
         file_id = file_open(libraries(i_library) % path, 'r')
         group_id = open_group(file_id, name)
-        call resonant_nuc % from_hdf5(group_id, '0K')
+        call resonant_nuc % from_hdf5(group_id, temperature, &
+             TEMPERATURE_NEAREST, 1000.0_8)
```

Comment:
Why is the tolerance 1000K?  Wouldn't that make this work for many temperatures, when we actually require that it is 0K?